third_party/mpi/BUILD

```diff
@@ -19,4 +19,5 @@ cc_library(
         "//conditions:default": ["libmpi.so"],
     }),
     hdrs = ["mpi.h", "mpi_portable_platform.h"],
```

Comment:
Correct, I have not enough experience with Bazel to let it determine at runtime/configuration time if the MPI implementation pointed to is OpenMPI or MPICH based. Therefore I just added it for the OpenMPI case which is what I used internally.